spec/lib/efile/nj/nj1040_calculator_spec.rb

```diff
@@ -265,4 +265,126 @@ def over_65_birth_year
       expect(instance.lines[:NJ1040_LINE_42].value).to eq(instance.lines[:NJ1040_LINE_39].value)
     end
   end
+  
+  describe 'line 65 - NJ child tax credit' do
+    context 'when taxpayer is married filing separately' do
+      let(:intake) { create(:state_file_nj_intake, :married_filing_separately) }
+      it 'returns nil' do
+        expect(instance.lines[:NJ1040_LINE_65].value).to eq(nil)
```

Comment:
[pebble] you're not actually doing `instance.calculate` here right so everything would be nil right?